src/pnpq/devices/odl_ozoptics_650ml.py

```diff
@@ -47,8 +48,14 @@ def set_step(self, value):
         response = self.serial_command(cmd)
         return response
 
-    def current_step():
-        pass
+    def get_step(self) -> int:
+        response = self.serial_command(f"S?")
```

Comment:
The "f-" in "f-string" means "format."

There is nothing to format in this string, therefore, it can just be a normal string.
```suggestion
        response = self.serial_command("S?")
```

Reply:
Done!